lib/paleta.rb

```diff
@@ -3,4 +3,13 @@
 require 'paleta/palette'
 
 module Paleta
+  @rmagick_available = begin
```

Comment:
I'm not sure—I generally avoid using class variables. If I remember correctly, the differences between the two show up when inheritance is at play. That shouldn't apply here, but I may need a refresher on when it is best to use each one. Is there a specific case you're concerned about?